index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+	<title>This converts temperatures</title>
+</head>
+<body>
+	<script src="https://fb.me/react-0.13.0.min.js" type="text/javascript"></script>
```

Comment:
Kinda? It was never meant to be used in production. It was deprecated right when I started learning. I believe the last straw was when babel included jsx transform by default. [Docs about deprecation](https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html)